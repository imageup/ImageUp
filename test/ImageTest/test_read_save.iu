function main() -> int
{

        image: a = read("./images/tiny.jpg");
        prints("Done reading image");
        save("test.jpg", a);
        prints("Done saving image");
        return 0;
}
