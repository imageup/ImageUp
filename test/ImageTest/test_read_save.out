Done reading image
Done saving image
